AdyenComponents/Boleto/BoletoComponent.swift

```diff
@@ -0,0 +1,233 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import Foundation
+import UIKit
+
+/// A component that provides a form for Boleto payment.
+public final class BoletoComponent: PaymentComponent, LoadingComponent, PresentableComponent, Localizable, Observer {
+    
+    public func stopLoading() {
+        formComponent.stopLoading()
+    }
```

Comment:
can you move this down, I think the convention is to start with `var`'s and `let`'s , then initializer, then functions